Cargo.toml

```diff
@@ -97,7 +97,7 @@ static_assertions = "1.1.0"
 async-recursion = "1.1.1"
 winnow = "0.7"
 uds_windows = "1.1.0"
-nix = { version = "0.29", default-features = false, features = [
+nix = { version = "0.29", git = "https://github.com/swick/nix.git", branch = "wip/sopeerpidfd", default-features = false, features = [
```

Comment:
This will be a version bump after your PR is merged (and hopefully released, although I can be easily bribe..err.. convinced to use git main) so best to split this into a separate commit.

Reply:
I intended for this to get removed before the PR is merged. Moved to a separate commit anyway.